### PR TITLE
Update lustre to latest

### PR DESCRIPTION
**Pull request by alyssa-sm**
This PR updates the DDN-Exascaler commit reference to the latest version released. 

This version change includes a few new disk types.
- hyperdisk-balanced
- hyperdisk-extreme 
- hyperdisk-throughput

Along with the new disk types, there were two new parameters.
- disk_iops
- disk_mbps

**Manual testing**
To test the new disks and parameters, I used `examples/pfs-lustre.yaml` and changed the defaults for lustre. Below is an example of testing to ensure `hyperdisk-throughput` was working as intended. After deploying, I verified it was successful on Pantheon. 
```
  - id: lustre
    source: community/modules/file-system/DDN-EXAScaler
    use: [network1]
    settings:
      local_mount: /lustre
      oss:
        node_type: "n2-standard-16"
        node_cpu: "Intel Cascade Lake"
        nic_type: "GVNIC"
        public_ip: true
        node_count: 1
      ost: 
        disk_bus: "SCSI"
        disk_type: "hyperdisk-throughput"
        disk_iops: null
        disk_mbps: 20
        disk_size: 2048
        disk_count: 1
        disk_raid: false
```
All the available disk types were tested in a similar fashion.

### CODE REVIEW

File: community/modules/file-system/DDN-EXAScaler/main.tf
Comment (nick-stroud) [inline community/modules/file-system/DDN-EXAScaler/main.tf:1]: Please run all lustre integration tests before merging.

File: community/modules/file-system/DDN-EXAScaler/README.md
Comment (nick-stroud) [inline community/modules/file-system/DDN-EXAScaler/README.md:1]: nit: It might be nice to provide an example blueprint snippet in this README similar to what you have in the PR description, showing how to use hyperdisk. You could also do this in a separate PR.

File: community/modules/file-system/DDN-EXAScaler/main.tf [hunk 1]
```diff
@@ -42,7 +42,7 @@ locals {
 }
 
 module "ddn_exascaler" {
-  source          = "github.com/DDNStorage/exascaler-cloud-terraform//gcp?ref=a3355d50deebe45c0556b45bd599059b7c06988d"
+  source          = "github.com/DDNStorage/exascaler-cloud-terraform//gcp?ref=fafd923"
```
Comment (nick-stroud) [inline community/modules/file-system/DDN-EXAScaler/main.tf:45]: nit: is there a reason to use the commit hash instead of the tagged version? Could we at least put the tagged version in a comment at the end of the line so we don't have to cross reference in the future:
```
...cloud-terraform//gcp?ref=fafd923"  # tag=scripts/x.x
```

**Review by nick-stroud — APPROVED** [review]

File: community/modules/file-system/DDN-EXAScaler/README.md
Comment (alyssa-sm) [reply community/modules/file-system/DDN-EXAScaler/README.md:1]: This is a good idea. I will handle this in a separate PR.

File: community/modules/file-system/DDN-EXAScaler/main.tf (hunk 1 above)
Comment (alyssa-sm) [reply community/modules/file-system/DDN-EXAScaler/main.tf]: I had trouble when I tried using `ref=scripts/2.2.0`. It didn't recognize that there was more after the `/` and I couldn't get it to work, so I decided to stick to the commit hash for now. I added the comment for the tag, so it's easy to find the reference in the future.
